api/inventory/agents.proto

```diff
@@ -154,6 +154,20 @@ message QANMySQLPerfSchemaAgent {
   string pmm_agent_id = 2;
   // Service identifier.
   string service_id = 3;
+  // MySQL username.
```

Comment:
```suggestion
  // MySQL username for scraping metrics.
```

Reply:
But it's not for scraping metrics! 😆

Reply:
Ok, so for what?? 😕

Reply:
We want to keep using the word "metrics" only for Prometheus metrics. QAN is about performance data / query statistics / performance events… Terminology is a bit in flux. But not "metrics".

Reply:
Oh, and "scraping" is very Prometheus-specific.

Reply:
It's confusing because all these things called "agents" ... We have an `agent` to all life situations 😄

Reply:
> An **Agent** represents something that runs on the Node which is not useful itself but instead provides insights (metrics, query performance data, etc) about Nodes and/or Services.

Reply:
Yep, I'm saying about this. It's a big and blurred abstraction I guess. I mean agent doesn't have a single responsibility. You can say that agents responsibility is `provide insights`, but it also very abstract and in fact, it can be different things with different business logic and meaning. So there is a reason I a little bit confusing sometimes.